gradle/libs.versions.yaml

```diff
@@ -0,0 +1,27 @@
+metadata:
+  format: {version: 1.0}
+
+dependencies:
+  # Runtime libraries
+  checker: "org.checkerframework:checker-qual:3.12.0"
```

Comment:
```suggestion
  checker-qual: "org.checkerframework:checker-qual:3.12.0"
```

Reply:
this would produce an accessor of `libs.checker.qual` -- do you want that, or something like `libs.checkerQual`?